core-bundle/contao/library/Contao/InsertTags.php

```diff
@@ -181,7 +181,7 @@ static function ($allowedTag)
 			// Skip certain elements if the output will be cached
 			if ($blnCache)
 			{
-				if ($elements[0] == 'date' || $elements[0] == 'form_session_data' || $elements[0] == 'fragment' || ($elements[1] ?? null) == 'referer' || strncmp($elements[0], 'cache_', 6) === 0)
+				if ($elements[0] == 'date' || $elements[0] == 'form_session_data' || $elements[0] === 'form_confirmation_message' || $elements[0] == 'fragment' || ($elements[1] ?? null) == 'referer' || strncmp($elements[0], 'cache_', 6) === 0)
```

Comment:
Can we maybe shorten the parameter name and insert tags? For example `form_message` and `$formMessage`?

Reply:
I have changed the insert tag to `{{form_message}}` in [2e6e817](https://github.com/contao/contao/pull/5307/commits/2e6e81781af40b4111db68285bf09cd12f9d89fc). I am not sure what parameter you mean for `$formMessage`, though.

Reply:
<img width="537" alt="" src="https://user-images.githubusercontent.com/1192057/196375171-625fe2d2-2229-4425-8e6d-e97f59812a3c.png">

😉

Reply:
Frankly, I think the `confirmationMessage` column name is more self-explanatory than just a `message`, because that – theoretically – could be of any type 🤔 Maybe then we could also change insert tag from `{{form_message}}` to `{{form_confirmation}}` …? What do you think?

Reply:
I like `{{form_confirmation}}`. Can we rename the variable to `$this->objModel->confirmation` then as well or would this be too unspecific again?

Reply:
Changed in https://github.com/contao/contao/pull/5307/commits/70fb99778529cbb7bfb4711ce9469b625c887191 :+1: